src/doc/palette.cpp

```diff
@@ -25,6 +25,14 @@ namespace doc {
 
 using namespace gfx;
 
+enum FitCriteria{
```

Comment:
Prefer a `enum class` for global enums (enums outside classes):
```suggestion
enum class FitCriteria {
```